src/components/atoms/OcResourceLink/OcResourceLink.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <component
+    :is="componentType"
+    v-bind="componentProps"
+    v-if="isResourceClickable"
+    :target="linkTargetBlank"
+    :aria-describedby="opensInNewWindowDescriptionId"
+    @click.stop="emitClick"
+    @click.native.stop
+  >
+    <slot :opensInNewWindowDescriptionId="opensInNewWindowDescriptionId" />
+  </component>
+  <span v-else>
+    <slot />
+  </span>
+</template>
+
+<script>
+import uniqueId from "../../../utils/uniqueId"
+
+/**
+ * Wraps content in a resource link
+ */
+export default {
+  name: "OcResourceLink",
+  status: "prototype",
+  release: "12.2.3",
```

Comment:
Please make this "unreleased" since we don't know what type the next release will be just yet 😉